blocks/init/src/Blocks/custom/featured-posts/featured-posts.php

```diff
@@ -32,7 +32,7 @@
 		];
 
 		if ($excludeCurrentPost) {
-			$args['post__not_in'] = $posts->ID;
+			$args['post__not_in'] = [ $post->ID ];
```

Comment:
Isn't the usage of `post__not_in` argument discouraged because of performance issues?

Reply:
yes but it is only one post so it is kinda ok